library/src/app/components/identity-verification/identity-verification.component.ts

```diff
@@ -161,6 +161,37 @@ export class IdentityVerificationComponent implements OnInit, OnDestroy {
     }
   }
 
+  /**
+   * Checks for an existing identity verification
+   *
+   * Handles most recent IDV that exists in a waiting state
+   * Creates new IDV if no IDV exists or the state is not waiting
+   **/
+  checkExistingIdentityVerifications(): void {
+    this.identityVerificationService
+      .listIdentityVerifications()
```

Comment:
Should we limit this to getting just `1` IDV?

Reply:
Sure. It's limited to `1` now